wmn-data.json

```diff
@@ -273,6 +273,23 @@
         "known" : ["zala", "lindapearl"],
         "cat" : "social"
        },
+       {
+        "name": "AniList",
+        "uri_check": "https://graphql.anilist.co",
+        "uri_pretty": "https://anilist.co/user/{account}"
+        "post_body": "{\"query\":\"query{User(name:\\\"{account}\\\"){id name}}\"}",
+        "e_code": 200,
+        "e_string": "\"id\":",
+        "m_code": 404,
+        "m_string": "Not Found",
+        "known": ["test", "johndoe"],
+        "cat": "social",
+        "protection": [],

```

Comment:
This is optional. If it is empty, please remove it.

Reply:
Removed it.